src/services/FirebaseService.ts

```diff
@@ -33,6 +34,16 @@ export class FirebaseService implements IFirebaseService {
         return result;
     }
 
+    public async getDapp(address: string, network: NetworkType): Promise<NewDappItem | undefined> {
```

Comment:
I see this function can be undefinable, and the only time this is called is at line `148` in `DappsStakingController.ts` from a direct HTTP request.

I recommend that we add proper input checking and handle null cases.

Reply:
Fixed in [this](https://github.com/AstarNetwork/astar-token-api/pull/47/commits/789835bc8a13e5f3b82fc1373b1d4e987076c121) commit 

None of parameters can't have null value, but address can be empty string. I added check for that. network can have only values specified in NetworkType so I didn't do any special check.